pkg/operator/starter.go

```diff
@@ -190,7 +214,6 @@ func RunOperator(ctx context.Context, kubeconfig *restclient.Config) error {
 	kubeInformersForOpenShiftConfig.Start(ctx.Done())
 	kubeInformersForOpenShiftConfigManaged.Start(ctx.Done())
 	kubeInformersForKubeSystem.Start(ctx.Done())
-	configInformers.Start(ctx.Done())
```

Comment:
let's try adding this line back here? I know it's already called higher up on this file, but let's just test and see if changes anything.

Reply:
add it multiple times or remove the one from above and add here?

Reply:
Add it multiple times